pkg/ansible/proxy/proxy_test.go

```diff
@@ -102,18 +105,12 @@ func createPod(name, namespace string, cfg *rest.Config) runtime.Object {
 			ActiveDeadlineSeconds: &three,
 		},
 	}
-	cl, err := client.New(cfg, client.Options{})
-	err = cl.Create(context.Background(), pod)
-	if err != nil {
-		return nil
-	}
+	cl, _ := client.New(cfg, client.Options{})
```

Comment:
Any reason you're removing these errors? Why not just check both?

Reply:
Because they have been not used. what is your suggestion?

Reply:
Accept a `*testing.T` as an argument to `createPod` and `t.Fatal` if there's a non-nil error.

Reply:
I should have left my review comment about this here, but didn't see it until I submitted my review. See https://github.com/operator-framework/operator-sdk/pull/2230#discussion_r347725796